docs/content/grafana/add_grafana_dashboard.md

```diff
@@ -27,11 +27,18 @@ Pick a suitable `metadata.name`, ensure that its unique in the [`grafana/overlay
 
 Save this file under `grafana/overlays/moc/smaug/dashboards/${DASHBOARD_FOLDER}/grafana-dashboard-example.yaml`.
 
+Make sure to create a kustomization inside your '${DASHBOARD_FOLDER}'
+
+```bash
+  cd ${DASHBOARD_FOLDER}
```

Comment:
just for consistency, can you add `$` signs before each command? or remove it from lines 40/41?